### DISCUSSION TITLE
Binary variables not working.

### DISCUSSION THREAD

**Discussion by davidcarails** [coin-or/Clp#233] [Q&A]
I am running this testcase lp_model.mps.gz with the clp standalone executable.
The solution values are not binary.
I get values like 0.25 and 0.5
The command I used is below
<pre>
[lp_model.mps.gz](https://github.com/coin-or/Clp/files/8627668/lp_model.mps.gz)
clp lp_model.mps.gz -maximize -Solve -Solu sol.txt
</pre>
[lp_model.mps.gz](https://github.com/coin-or/Clp/files/8627668/lp_model.mps.gz)

**Comment by jjhforrest** [comment]
Clp only solves linear problems not integer ones.  You need to use cbc.
